### ISSUE TITLE
Need to update? Translation: Chinese

### ISSUE THREAD

**Issue by longjj** [jwasham/coding-interview-university#243]
Hi, @jwasham 

The README is helpful to me! I have also read the Chinese version while I found that the Chinese version seemed to be a little out of date. The content is similar while the structure and some details are different between the English and Chinese version since the Chinese version is still focusing on becoming a future Googler.

So I want to know if there is a  necessity to update the Chinese version from google-interview-university to coding-interview-university version? I have interest in this work.

Thx.

**Comment by Abson** [comment]
As far as i see，the translation version of chinese that have a lot of error of syntax

**Comment by longjj** [comment]
Once I found the Chinese version is out of date, I mainly refer to the newest English version.

In my opinion, the current version of Chinese may have errors of syntax, while the main study framework has been displayed generally. If it is needed we can translate it at the beginning, but it will cost more time, and I think it is unnecessary to redo the same work.

May be we can cooperate.

**Comment by jwasham** [comment]
Feel free to update the translation!

**Comment by longjj** [comment]
OK, I will spend specific time on this task every weekend. Once I finish I will pull requests.

**Comment by longjj** [comment]
@jwasham 

I recently contacted the original translator. I decide to update it from the original Chinese version. So once we finish updating the version, the original translator will merge my changes and he will send pull requests to the repo.

**Comment by longjj** [comment]
@jwasham 
I am really sorry that I can hardly finish the translation perfectly because of my heavy college projects. I have finished my first draft recently but I found that there are still lots of Chinese translation rules needing to be fixed to meet the original Chinese translation organization's requirement. 

In this year I have to spend most of my time applying for the postgraduate program.  I won't translate it in a long period of time. I consider the current Chinese version better than my first draft, so I may close this issue.  

Maybe when I have free time again, I will reconsider to continue my work if no one updates the current version.

Sorry again :(
